components/Hero/styles.css

```diff
@@ -45,6 +45,20 @@
   z-index: 1;
 }
 
+.Hero-links {
+  position: relative;
+  max-width: 1024px;
+  height: 40px;
+  margin-top: -45px;
+}
```

Comment:
spoke with mel this morning and if by constrained you mean constrained to the 1024px, then yes it is also supposed to be.